.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+Your PR title should start with one of the following conventions:
+
+- build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- ci: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+- docs: Documentation only changes
+- feat: A new feature
+- fix: A bug fix
+- perf: A code change that improves performance
+- refactor: A code change that neither fixes a bug nor adds a feature
+- style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- test: Adding missing tests or correcting existing tests
+
+Other things to note:
+
+- Please describe the change using verbs (ex: Removes X from Y, Creates new Z)
+- Add any relevant unit / integration tests, and possibly instructions for how to reproduce
```

Comment:
nit: In a separate line: "- Changes will be previewable via vercel. Non-obvious changes should include instructions for how to reproduce them"